x-pack/plugins/ui_actions_enhanced/public/dynamic_actions/dynamic_action_manager.ts

```diff
@@ -84,7 +86,10 @@ export class DynamicActionManager {
         return actionDefinition.isCompatible(context);
       },
     });
-    for (const trigger of triggers) uiActions.attachAction(trigger as any, actionId);
+
+    // TODO: https://github.com/elastic/kibana/issues/59569
+    // for (const trigger of triggers) uiActions.attachAction(trigger as any, actionId);
+    uiActions.attachAction(APPLY_FILTER_TRIGGER, actionId);
```

Comment:
@streamich, this is a place where I did "runtime migration" to make dashboard drilldowns created before work after this change. It relies, that we don't 

Maybe you have thoughts on better place? 

----
I assume that in #59569, each drilldown would explicitly specify triggers it supports and then we could add migrate function to drilldown definition which would allow to migrate both triggers and action config.

Reply:
Agreed offline to put this logic into storage implementation.